pkg/operator/hive/hive_controller.go

```diff
@@ -223,6 +261,41 @@ func (r *ReconcileHiveConfig) Reconcile(request reconcile.Request) (reconcile.Re
 		Namespace: hiveNamespace,
 	})
 
+	if r.syncAggregatorCA {
+		// The reason we use the configmap lister here and not the regular client is that this controller is only watching
+		// resources in the 'hive' namespace. Trying to fetch from a different namespace results in a NotFound error.
+		aggregatorCAConfigMap, err := r.managedConfigCMLister.ConfigMaps(managedConfigNamespace).Get(aggregatorCAConfigMapName)
```

Comment:
Another brevity nit: "We use the configmap lister and not the regular client to only watch resources in the hive namespace."